addons/mail/models/mail_render_mixin.py

```diff
@@ -38,6 +39,25 @@ def format_time(env, time, tz=False, time_format='medium', lang_code=False):
         return time
 
 
+template_env_globals = {
```

Comment:
Unecessary diff, would keep the one from tools.jinja (that could be renamed if you want)